.github/workflows/build-on-release.yml

```
@@ -62,81 +174,142 @@ jobs:
+
+    - name: Clean up keychain and provisioning profile
+      if: always()
+      run: |
+        security delete-keychain $RUNNER_TEMP/app-signing.keychain-db
+        rm ~/Library/MobileDevice/Provisioning\ Profiles/build_pp.mobileprovision
+
+  build-macos-intel:
+    runs-on: macos-15-intel
+
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)